docs/fsharp/language-reference/attributes.md

```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
   </tr>
```

Comment:
should do a similar change on line 67 as well

Reply:
Which similar change?